index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="sft-n-hrd.js"></script>
+    <title>sft n hrd</title>
+    <meta name="description" content="A simple website to play with curious measurements.">
+  </head>
+  <body>
+    <h1>sft n hrd</h1>
+
```

Comment:
all these changes will be present in the other PR. You can merge master into this branch and these changes will do away